default.nix

```diff
@@ -10,7 +10,7 @@
   # https://github.com/NixOS/nix/issues/3539 ).
   nvidiaVersionFile ? null,
   # Enable 32 bits driver
-  # This is one by default, you can switch it to off if you want to reduce a
+  # This is ne by default, you can switch it to off if you want to reduce a
```

Comment:
Is that a typo?
```suggestion
  # This is on by default, you can switch it to off if you want to reduce a
```

Reply:
🤣 this file is cursed. Thanks for merging :)